xinference/web/ui/src/scenes/launch_model/Text2ImageCard.js

```diff
@@ -0,0 +1,337 @@
+      transition: 'transform 0.2s ease-in-out',
+    },
+    iconRow: {
+      display: 'flex',
+      justifyContent: 'space-between',
+      alignItems: 'center',
+    },
+    iconItem: {
+      display: 'flex',
+      flexDirection: 'column',
+      alignItems: 'center',
+      margin: '20px',
+    },
+    boldIconText: {
+      fontWeight: 'bold',
+      fontSize: '1.2em',
+    },
+    muiIcon: {
+      fontSize: '1.5em',
+    },
+    smallText: {
+      fontSize: '0.8em',
+    },
+    langRow: {
+      margin: '2px 5px 40px 5px',
+    },
+  }
+
+  const handeCustomDelete = (e) => {
+    e.stopPropagation()
+    fetcher(url + `/v1/model_registrations/image/${modelData.model_name}`, {
+      method: 'DELETE',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    })
+      .then(() => setCustomDeleted(true))
+      .catch(console.error)
+  }
+
+  // Set two different states based on mouse hover
+  return (
+    <Box
+      style={hover ? styles.containerSelected : styles.container}
+      onMouseEnter={() => setHover(true)}
+      onMouseLeave={() => setHover(false)}
+      onClick={() => {
+        if (!selected && !customDeleted) {
+          setSelected(true)
+        }
+      }}
+    >
+      {/* First state: show description page */}
+      <Box style={styles.descriptionCard}>
+        <div style={styles.titleContainer}>
+          {is_custom && (
+            <Stack
+              direction="row"
+              justifyContent="space-evenly"
+              alignItems="center"
+              spacing={1}
+            >
+              <h2 style={styles.h2}>{modelData.model_name}</h2>
+              <IconButton
+                aria-label="delete"
+                onClick={handeCustomDelete}
+                disabled={customDeleted}
+              >
+                <DeleteIcon />
+              </IconButton>
+            </Stack>
+          )}
+          {!is_custom && <h2 style={styles.h2}>{modelData.model_name}</h2>}
+          <Stack
+              spacing={1}
+              direction="row"
+              useFlexGap
+              flexWrap="wrap"
+              sx={{ marginLeft: 1 }}
+          >
+              {(() => {
+                      return <Chip label={modelData.model_family} variant="outlined" size="small" />
+              })()}
+              {(() => {
+                  if (modelData.is_cached) {
+                      return <Chip label="Cached" variant="outlined" size="small" />;
+                  }
+              })()}
+              {(() => {
+                  if (is_custom && customDeleted) {
+                      return <Chip label="Deleted" variant="outlined" size="small" />;
+                  }
+              })()}
+          </Stack>
+        </div>
+        {hover ? (
+          <p style={styles.instructionText}>
+            Click with mouse to launch the model
+          </p>
+        ) : (
+          <p style={styles.instructionText}></p>
+        )}
```

Comment:
Change to this:
```
{!selected && hover && (
          <p style={styles.instructionText}>
            Click with mouse to launch the model
          </p>
        )}
```
May be you need to rebase to the latest main code. You are using the old version code here.

Reply:
Alright, I'll make the change.